### PR TITLE
Change ID assertion endpoint mode to CORS

### PR DESCRIPTION
**Pull request by npm1**
Fixes https://github.com/fedidcg/FedCM/issues/428

***
<a href="https://pr-preview.s3.amazonaws.com/fedidcg/FedCM/pull/547.html" title="Last updated on Aug 1, 2024, 9:06 PM UTC (61a4597)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/fedidcg/FedCM/547/955d4d0...61a4597.html" title="Last updated on Aug 1, 2024, 9:06 PM UTC (61a4597)">Diff</a>

### CODE REVIEW

File: spec/index.bs [hunk 1]
```diff
@@ -2529,6 +2525,16 @@ credentialed requests it receives, which ensures that the request was initiated
...
 
+<!-- ============================================================ -->
+## CORS Header ## {#sec-cors-header}
+<!-- ============================================================ -->
+
+The FedCM API allows the response from the [=identity assertion endpoint=] to be shared to the
+[=RP=]. Because of this, we impose the requirement that the [=IDP=] explicitly consents to this
+sharing taking place by using the "cors" [=request/mode=] when fetching this endpoint. This also
+helps with servers that may accidentally ignore the <a http-header>Sec-Fetch-Dest</a>: they cannot
+ignore CORS, as otherwise the fetch will fail.
```
Comment (TallTed) [inline spec/index.bs:2536]:
```suggestion
ignore CORS, as without it the fetch will fail.
```

**Review by samuelgoto — APPROVED** [review]
LGTM

**Review by bvandersloot-mozilla — APPROVED** [review]

**Comment by npm1** [comment]
Thanks for the review, will merge after the repo moves to the WG, per instructions on the call.

**Review by timcappalli — APPROVED** [review]
LGTM

**Review by samuelgoto — APPROVED** [review]
LGTM++

**Pull request merged**